x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/notification/email/attachment/HttpEmailAttachmentParser.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I think you need to remove the corresponding file with the misspelled class name? Possibly you can move the file with git so that we don't lose history around it? You can also do this in a separate smaller PR.

Reply:
*sigh*, I forgot that converting hg moves to git sometimes results in this mess, I'll fix.